src/Microsoft.AspNet.OData/Adapters/WebApiRequestMessage.cs

```diff
@@ -26,7 +26,7 @@ internal class WebApiRequestMessage : IWebApiRequestMessage
         /// <summary>
         /// The inner request wrapped by this instance.
         /// </summary>
-        private HttpRequestMessage innerRequest;
+        internal HttpRequestMessage innerRequest;
```

Comment:
>internal HttpRequestMessage innerRequest; [](start = 8, length = 41)

expose it throw an internal property

Reply:
AspNet Core `innerRequest` is also `internal`. So i wanted some consistency.

Reply:
Can we change both? internal field is not good design

---
In reply to: [501473795](https://github.com/OData/WebApi/pull/2311#discussion_r501473795) [](ancestors = 501473795)

Reply:
#Resolved